osbuild/main_cli.py

```diff
@@ -148,6 +148,10 @@ def osbuild_cli(*, sys_argv):
         sys.stdout.write("\n")
         return 0
 
+    if args.output_directory == None and args.checkpoint == None:
+        print("No output directory or checkpoints specified, exited without building.")
```

Comment:
Nit: you can just use `not args.output_directory` and `not args.checkpoint` instead of the manual `== None` check.

Reply:
NB: `== None` might invoke the class' `__eq__` method while `is` just compares object identity (a la `id(a) == id(b)`), so normally the latter is preferred unless one really meant the `== None`, like with `NumPy` arrays. But here just use the "truthiness" of the object itself, no need for explicit `None` checks.

Reply:
fixed, thanks!